sys/net/gnrc/network_layer/ipv6/nib/Kconfig

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_GNRC_IPV6_NIB
+    bool "Configure GNRC IPv6 NIB"
+
+if KCONFIG_MODULE_GNRC_IPV6_NIB
+
+config GNRC_IPV6_NIB_6LBR
+    bool "6LoWPAN border router features"
+    default y if MODULE_GNRC_IPV6_NIB_6LBR
+    help
+        Enable 6LoWPAN Border Router (6LBR) features.
+        @see https://tools.ietf.org/html/rfc6775#section-2
+
+config GNRC_IPV6_NIB_6LR
+    bool "6LoWPAN router features"
+    default y if MODULE_GNRC_IPV6_NIB_6LR || GNRC_IPV6_NIB_6LBR
+    help
+        Enable 6LoWPAN Router (6LR) features.
+        @see https://tools.ietf.org/html/rfc6775#section-2
+
+config GNRC_IPV6_NIB_6LN
+    bool "6LoWPAN node features"
+    default y if MODULE_GNRC_IPV6_NIB_6LN || GNRC_IPV6_NIB_6LR
+    help
+        Enable 6LoWPAN Node (6LN) features.
+        @see https://tools.ietf.org/html/rfc6775#section-2
+
+config GNRC_IPV6_NIB_ROUTER
+    bool "Router features"
+    default y if MODULE_GNRC_IPV6_NIB_ROUTER || GNRC_IPV6_NIB_6LR
+
+config GNRC_IPV6_NIB_SLAAC
+    bool "Stateless address auto-configuration"
+    default y if MODULE_GNRC_IPV6_NIB_6LBR
+    default n if MODULE_GNRC_IPV6_NIB_6LR || MODULE_GNRC_IPV6_NIB_6LN
+    default y
+
+config GNRC_IPV6_NIB_QUEUE_PKT
+    bool "Use packet queue with address resolution"
+    default n if MODULE_GNRC_IPV6_NIB_6LN || GNRC_IPV6_NIB_6LN
+    default y
+
+config GNRC_IPV6_NIB_ARSM
+    bool "Use classic NDP address resolution state-machine"
+    default n if MODULE_GNRC_IPV6_NIB_6LN && !GNRC_IPV6_NIB_6LR
+    default y
+
+config GNRC_IPV6_NIB_DNS
+    bool "Support for DNS configuration options"
+    default y if MODULE_GNRC_IPV6_NIB_DNS
+
+config GNRC_IPV6_NIB_ADV_ROUTER
+    bool "Activate router advertising at interface start-up"
+    default y if GNRC_IPV6_NIB_ROUTER && (!GNRC_IPV6_NIB_6LR || GNRC_IPV6_NIB_6LBR)
+
+config GNRC_IPV6_NIB_DC
+    bool "Destination cache"
+
+config GNRC_IPV6_NIB_MULTIHOP_P6C
+    bool "Multihop prefix and 6LoWPAN context distribution"
+    default y if GNRC_IPV6_NIB_6LR
+
+config GNRC_IPV6_NIB_NO_RTR_SOL
+    bool "Disable router solicitations"
+    help
+        @warning Only set this if you know what you're doing.
```

Comment:
Could you add a warning like you did for `GNRC_IPV6_NIB_ABR_NUMOF` here as well? Something like

> Disabling router solicitations will make your host have to wait longer for router advertisements. This option is only meant for testing and experimentation and should not be activated in production.

Reply:
Sure!